### PR TITLE
Added description and use cases for zero-shot classification

### LINKED ISSUE #361: Zero-shot-classification Task Page
This is the tracker for completion of task page for `zero-shot-classification` task.

List of items to contribute:
- [ ] Task description
- [ ] Subtasks (if there is any)
- [ ] Most used dataset for the task
- [ ] Most used model for the task
- [ ] Metrics that are used to evaluate the task
- [ ] Libraries used for the task
- [ ] Use cases

### PR DESCRIPTION
**Pull request by 02shanks**
fixed #361 by adding description and use cases

### CODE REVIEW

**Comment by 02shanks** [comment]
I am making new PR

File: tasks/src/Zero-Shot_Image_Classification/data.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "",
+		},
+	],
+	summary:      "Zero-shot image classification is a task in computer vision where the aim is to classify images into one of several pre-defined categories, without any prior training or knowledge of the categories.",
```
Comment (merveenoyan) [inline tasks/src/Zero-Shot_Image_Classification/data.ts:25]: It's good to keep PRs minimal for the sake of review process, so I'd encourage you to remove the files that aren't about `zero-shot-classification` (also note that we already have someone contributed for this and given this is PR for zero-shot-classification)

File: tasks/src/Zero-shot-classification/about.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Use Cases
+
+### Text Classification
+
+Zero-shot text classification is a task in natural language processing (NLP) where a model is trained on a set of labeled examples but is then able to classify new examples from previously unseen classes by transferring knowledge from seen to unseen classes.
```
Comment (merveenoyan) [inline tasks/src/Zero-shot-classification/about.md:5]: I loved the definition!

File: tasks/src/Zero-shot-classification/data.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	],
+	
+	models: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "i",
```
Comment (merveenoyan) [inline tasks/src/Zero-shot-classification/data.ts:22]:
```suggestion
			id:          "",

```
I think this is a mistake right?

File: tasks/src/Zero-shot-classification/data.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+		},
+	],
+	summary:      "Zero-shot classification is a machine learning classification task where the training data does not contain any labels for the classes that the model will be asked to predict. The model must learn to map the input data to the correct class based solely on the input data. This is in contrast to traditional classification tasks, where the model is trained on data that is labeled with the correct class.",
```
Comment (merveenoyan) [inline tasks/src/Zero-shot-classification/data.ts:25]: Can you simplify the language here a little? 🙂

> Zero-shot classification is a machine learning classification task where the training data does not contain any labels for the classes that the model will be asked to predict.

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot for working on this!!! Left a very general feedback, can you check? 🙂

**Comment by 02shanks** [comment]
@merveenoyan  I messed up with this PR(committed every change on this). Can you review #385 for the reinforcement task and #386 for the zero-shot-classification? Sorry for the inconvenience.

**Pull request closed without merging**